### PR TITLE
doc: fix machine-specific capabilities leaking

### PR DESCRIPTION
**Pull request by ivan770**
# Context

Closes: #9251

# Priorities

Add :+1: to [pull requests you find important](https://github.com/NixOS/nix/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc).

### CODE REVIEW

**Comment by Ericson2314** [comment]
21520297dad8a198004e4551808a4818a8de7ddd did the same solution

Can you add comments to both saying it is for reproducibility of the manual? The `{}, false` isn't so clear to the reader on its own :).

**Comment by Ericson2314** [comment]
OK to do this in a follow up PR, but we should also test the the linux-built and mac-built manuals are identical.

**Comment by ivan770** [comment]
> [2152029](https://github.com/NixOS/nix/commit/21520297dad8a198004e4551808a4818a8de7ddd) did the same solution
> 
> Can you add comments to both saying it is for reproducibility of the manual? The `{}, false` isn't so clear to the reader on its own :).

The `documentDefault = false` value was missing for this particular setting. About the documentation - should I add to other sections where `{}, false` snippet is used?

File: src/libstore/store-api.hh [hunk 1]
```diff
@@ -167,7 +167,7 @@ struct StoreConfig : public Config
           Optional [system features](@docroot@/command-ref/conf-file.md#conf-system-features) available on the system this store uses to build derivations.
 
           Example: `"kvm"`
-        )" };
+        )", {}, false};
```
Comment (Ericson2314) [inline src/libstore/store-api.hh:170]: something like this
```suggestion
        )",
        {},
        // default would like properties specific to this machine into the manual
        false,
    };
```

**Comment by Ericson2314** [comment]
> should I add it to other sections where {}, false snippet is used?

**Comment by Ericson2314** [comment]
Thanks!

**Comment by tomberek** [comment]
Running into this same issue with ssl-cert-file, would we want to set a default?

**Pull request merged**